### DISCUSSION TITLE
libpcap.so.0.8: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by BetterDefender** [projectdiscovery/naabu#238] [Q&A] [answered]
The following problems occurred when running naabu on Centos 7 system. I tried to solve it but failed. I need technical support, thank you!

Error log：
./naabu: error while loading shared libraries: libpcap.so.0.8: cannot open shared object file: No such file or directory
![image](https://user-images.githubusercontent.com/59255707/139060593-1b2e5a88-d25e-48e2-bb5a-6013e5fffff5.png)

**Comment by hkzck** [comment]
me too,
./naabu: error while loading shared libraries: libpcap.so.0.8: cannot open shared object file: No such file or directory

![image](https://user-images.githubusercontent.com/48936012/177913647-2f01bd2c-d6b7-4c1f-9bfe-f982fa3d82db.png)

![image](https://user-images.githubusercontent.com/48936012/177913609-a4c633a2-77dc-4fbf-95ca-26d26b9e205d.png)

**Comment by ehsandeep** [comment]
@hkzck could you retry with latest version of naabu and share the output of `naabu -hc`?

**Comment by mclang** [comment, marked as answer]
Found out this old issue while troubleshooting the same problem.

Trying `naabu -hc` gave the same error.

I'm using Arch based Endeavour OS and do have `libpcap` installed, but the file `naabu` needs was not present:
```
$ find /usr/lib/ -name 'libpcap.so*'
/usr/lib/libpcap.so
/usr/lib/libpcap.so.1
/usr/lib/libpcap.so.1.10.3
```

Solution was to make symbolic link:
```
$ sudo ln -s /usr/lib/libpcap.so /usr/lib/libpcap.so.0.8
```

I hope this helps.

**Reply by ocervell** [reply]
Same here on Alpine ;) Thanks for the solution, it saved me !

**Comment by sec13b** [comment]
with ln -s /usr/lib/libpcap.so

~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ln -s /usr/lib/libpcap.so /usr/lib/libpcap.so.0.6
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

you will get something like that:
~~~~~~~~~~~~~~~~~~~~~~~~~~~
naabu: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by naabu)
naabu: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by naabu)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

you can use : 
go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest

or 
git clone https://github.com/projectdiscovery/naabu.git
cd naabu
cd v2
make


on centos 7 dont work.
